python/tchannel/tornado/tornado_req_handler.py

```diff
@@ -0,0 +1,92 @@
+import json
+from tornado import httputil
+from tornado.httputil import RequestStartLine
+from ..req_handler import RequestHandler
+
+
+class TornadoRequestHandler(RequestHandler):
+    def __init__(self, app):
+        self.request_callback = app
+
+    def start_serving(self, request_conn):
+        return _ServerRequestAdapter(self, request_conn)
+
+    def handle_request(self, context, conn):
+        """Handle incoming request
+
+        :param context: incoming message context
+        :param conn: incoming connection
+        """
+        request_delegate = self.start_serving(conn)
+        message = context.message
+        # process http message
+        if message.headers["as"] == "http":
+            method = "GET"
+            if (hasattr(message, "arg_3") and
+                    message.arg_3 is not None and
+                    message.arg_3 != ""):
```

Comment:
If you assert that the message is a `CallRequestMessage` then the `hasattr` check isn't needed and the other two are consolidated into `if message.arg_3:`